hooks/tk-multi-loader2/basic/scene_actions.py

```diff
@@ -113,6 +114,23 @@ def generate_actions(self, sg_publish_data, actions, ui_area):
                 }
             )
 
+        if "import_with_options" in actions:
+            vred_version = int(self.parent.engine.vred_version.replace(".", ""))
+            if vred_version >= 202210:
+                action_instances.append(
+                    {
+                        "name": "import_with_options",
+                        "params": None,
+                        "caption": "Open Import Dialog to change options...",
+                        "description": "This will open the Import Options Dialog.",
+                    }
+                )
+            else:
+                self.logger.debug(
+                    "Not able to add import_with_options to Loader actions."
+                )
+                self.logger.debug("This capability requires VRED 2022.1 or later.")
```

Comment:
We're hard coding this functionality to require version >=2022.1, would we want to consider a config setting instead, incase this changes for whatever reason in the future?

Reply:
For this case, I don't think so as we just want to check on a minimum required for the API import.